### DISCUSSION TITLE
Disable Imaginary's PDF Preview

### DISCUSSION THREAD

**Discussion by n-aggelidis** [nextcloud/all-in-one#2270] [Questions] [answered]
Hello there!

I would like to use Imaginary for all other file types except PDF. Is there a setting to not let it generate PDF previews?

Thank you for your help!

**Comment by szaimen** [comment, marked as answer]
Hi, there is no setting to disable this currently afaik. This would be a feature request for https://github.com/nextcloud/server/issues
